deploy/policy/templates/conjur-secrets.template.sh.yml

```diff
@@ -8,6 +8,7 @@ cat << EOL
   body:
     - &variables
       - !variable test_secret
+      - !variable test_secret2
```

Comment:
i don't like the `2` very much as it has no meaning. can we change it to something like `another_test_secret` to show its purpose?